stytch/b2b/organizations/members/types.go

```diff
@@ -47,6 +47,9 @@ type CreateParams struct {
 	// step if the Organization's MFA policy is set to `REQUIRED_FOR_ALL`.
 	MFAEnrolled bool `json:"mfa_enrolled,omitempty"`
 }
+type DangerouslyGetParams struct {
+	MemberID string `json:"member_id,omitempty"`
+}
```

Comment:
Can we add some docs for this method? It's okay if we don't do it in this PR, but make a task for it to track.